integration-tests/src/test/scala/play/AkkaServerProvider.scala

```diff
@@ -83,7 +84,7 @@ trait AkkaServerProvider extends BeforeAfterAll {
     new HttpsConnectionContext(context)
   }
 
-  def clientHttpsContext() = {
+  def clientHttpsContext(akkaSslConfig: Option[AkkaSSLConfig] = None): HttpsConnectionContext = {
```

Comment:
prefer overloads than default values? (bin compat)

Reply:
This is in testing machinery. So does not really matter.